src/ONE.Models.CSharp/Common/Configuration/Spreadsheet/ColumnConfig.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace ONE.Models.CSharp.Models.Common.Configuration.Spreadsheet
```

Comment:
Additionally Common.Configuration.Spreadsheet is very confusing as that encompasses to different MSApps in two different domains.

Not sure where to put these models really but this organization does not feel great.  Currently all of the protobuf models are in the One.Models.CSharp namespace.  Perhaps these models could also reside there or perhaps they should be in One.Models.CSharp.SDK or One.Models.CSharp.Extras or something similar.

Reply:
The purpose of this is to have a common model for the deserialization of spreadsheet configurations.
@AQI-DanG : how would you suggest namespacing configuration models that apply to different domains?

Reply:
This is also a json deserialization.  I would suggest either the above mentioned Json or JsonTypes namespace or add it to the protobuf library.  The term configuration is already pretty loaded so while applicable it might be best to steer clear of it if we can.

Reply:
Moved to Imposed.